nd4j/nd4j-backends/nd4j-tests/src/test/java/org/nd4j/autodiff/opvalidation/LayerOpValidation.java

```diff
@@ -634,8 +634,8 @@ public void testSeparableConv2dBasic() {
                 .dataFormat(Conv2DConfig.NCHW)
                 .build();
 
-        SDVariable out = sd.cnn().sconv2d(vars, c);
-        out = sd.nn().tanh("out", out);
+        SDVariable out = sd.cnn().separableConv2d(in, dW, pW, b, c);
+        out = sd.f().tanh(out);
```

Comment:
don't use f() in tests - again, this is going away soon
There's a whole lot in this test class.

If we _can't_ use one of the namespaces instead of f(), that means there's a problem that needs to be fixed (i.e., breaking API change)